### DISCUSSION TITLE
Which version of OCL is best for Sketchup 8 ?

### DISCUSSION THREAD

**Discussion by shodanx2** [lairdubois/lairdubois-opencutlist-sketchup-extension#632] [Q&A] [answered]
Hi,

I use only Sketchup 8.0.16846 and I would like to know, what is the best and most recent version of opencutlist that might run on my version ?

Thanks !

**Comment by bbeaulant** [comment, marked as answer]
Hi,

None. Sorry. SketchUp 8 is so much outdated.

- OpenCutList 0.x to 3.x runs on SketchUp 2014+
- OpenCutList 4.x to 6.x runs on SketchUp 2017+
